### ISSUE TITLE
Trident pelvis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7157]
**Preferred term label:**
Trident pelvis

**Synonyms**
Trident sign

**Definition (free text, please give PubMed ID)**
 Spurs at the medial and lateral acetabular margin and in the center of the acetabulum gives rise to shape resembling a three-pronged spear known as trident
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5510317/

-- update other related definitions

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619479 
PMID: 31816441  1/1 
PMID: 29138412  1/4 -- 
trident ilia with spikes at the sacrosciatic notches

**Your nano-attribution (ORCID)**
